lib/linguist/languages.yml

```diff
@@ -1182,6 +1182,15 @@ GAS:
   tm_scope: source.assembly
   ace_mode: assembly_x86
 
+GCC machine description:
```

Comment:
Read these and weep: [IRC log](https://github.com/github/linguist/blob/5066f66dcd7be1f8d68cecd1f226e2b4c517963b/lib/linguist/languages.yml#L1612), [OpenRC runscript](https://github.com/github/linguist/blob/5066f66dcd7be1f8d68cecd1f226e2b4c517963b/lib/linguist/languages.yml#L2618), [Python traceback](https://github.com/github/linguist/blob/5066f66dcd7be1f8d68cecd1f226e2b4c517963b/lib/linguist/languages.yml#L3031), [Raw token data](https://github.com/github/linguist/blob/5066f66dcd7be1f8d68cecd1f226e2b4c517963b/lib/linguist/languages.yml#L3146).